src/components/challenge-card/ASChallengeCard.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { Image, Text, View } from 'react-native'
+import { styles } from './asChallengeCard-styles'
+
+const PLAY_BUTTON = require('../../assets/icons/audio.png')
+const COMPLETED_TICK = require('../../assets/icons/complete.png')
+const BOOKMARK = require('../../assets/icons/marked-favourite.png')
+const REMOVE_BOOKMARK = require('../../assets/icons/not-marked-favourite.png')
+const REMINDER = require('../../assets/icons/reminder.png')
```

Comment:
use mapping for this like in assignment-1 and also move these constants to constant file (Make a dashboard-constants.ts)